docs/_tutorials/getting_started.md

```diff
@@ -13,3 +13,344 @@ redirect_from:
+
+Once you’ve installed a development proxy, run it to begin forwarding requests to a specific port (we’re using port `3000` for this example, but if you customized the port used to initialize your app use that port instead):
+
+```shell
+ngrok http 3000
+```
+
+![Running ngrok](../assets/ngrok.gif "Running ngrok")
+
+The output should show a generated URL that you can use (we recommend the one that starts with `https://`). This URL will be the base of your request URL, in this case `https://8e8ec2d7.ngrok.io`.
+
+---
+</details>
+
+Now you have a public-facing URL for your app that tunnels to your local machine. The Request URL that you use in your app configuration is composed of your public-facing URL combined with the URL your app is listening on. By default, Bolt apps listen at `/slack/events` so our full request URL would be `https://8e8ec2d7.ngrok.io/slack/events`.
+
+> ⚙️Bolt uses the `/slack/events` endpoint to listen to all incoming requests (whether shortcuts, events, or interactivity payloads). When configuring endpoints within your app configuration, you'll append `/slack/events` to all request URLs.
+
+Under the **Enable Events** switch in the **Request URL** box, go ahead and paste in your URL. As long as your Bolt app is still running, your URL should become verified.
+
+<!-- TODO - Should this be done as a list? -->
+After your request URL is verified, scroll down to **Subscribe to Bot Events**. There are four events related to messages: `message.channels` (listens for messages in public channels), `message.groups` (listens for messages in private channels), `message.im` (listens for messages in the App Home/DM space), and `message.mpim` (listens for messages in multi-person DMs).
+
+If you want your bot to listen to messages from everywhere it is, choose all four message events. After you’ve added the events you want your bot to listen to, click the green **Save Changes** button.
+
+---
+
+### Listening and responding to a message
+Your app is now ready for some logic. Let's start by using the `message()` method to attach a listener for messages.
+
+The following example listens to all messages that contain the lower-case word "hello" and responds with "Hey there @user!"
+
+<!-- TODO - Suggestions on the `reply_to_hello` function name? `message_hello`? -->
+```python
+import os
+from slack_bolt import App
+
+# Initializes your app with your bot token and signing secret
+app = App(
+  token=os.environ.get("SLACK_BOT_TOKEN"),
+  signing_secret=os.environ.get("SLACK_SIGNING_SECRET")
+)
+
+# Listens to incoming messages that contain "hello"
+@app.message("hello")
+def reply_to_hello(message, say):
+    # say() sends a message to the channel where the event was triggered
+    say(f"Hey there <@{message['user']}>")
```

Comment:
Just to be compatible with `responds with "Hey there @user!"` in the guide.
```suggestion
    say(f"Hey there <@{message['user']}>!")
```

Reply:
Incredibly observant and great catch @seratch! Fixed in 0c850c2a73a86d0323ec3d01215fa0ac43f7f498